backend/apps/ai/management/commands/ai_create_slack_message_chunks.py

```
@@ -0,0 +1,92 @@
+
+        self.openai_client = openai.OpenAI(api_key=openai_api_key)
+
+        total_messages = Message.objects.count()
+        self.stdout.write(f"Found {total_messages} messages to process")
+
+        batch_size = 100
+        for offset in range(0, total_messages, batch_size):
+            Chunk.bulk_save(
+                [
+                    chunk
+                    for message in Message.objects.all()[offset : offset + batch_size]
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)